src/main/java/org/entando/kubernetes/service/digitalexchange/component/EntandoBundleComponentUsageService.java

```diff
@@ -24,20 +27,27 @@
 import org.entando.kubernetes.model.entandocore.EntandoCoreComponentUsage;
 import org.entando.kubernetes.model.entandocore.EntandoCoreComponentUsage.IrrelevantComponentUsage;
 import org.entando.kubernetes.model.entandocore.EntandoCoreComponentUsageRequest;
+import org.entando.kubernetes.model.job.ComponentDataEntity;
 import org.entando.kubernetes.model.job.EntandoBundleComponentJobEntity;
+import org.entando.kubernetes.repository.ComponentDataRepository;
 import org.springframework.stereotype.Service;
 
 @Service
 @Slf4j
 public class EntandoBundleComponentUsageService {
 
+    public static final String APP_BUILDER_COMPONENT_SUBTYPE = "app-builder";
```

Comment:
Please remove this constant in favore of `WidgetDescriptor.TYPE_WIDGET_APPBUILDER`

Reply:
done